vm/safepoints.cpp

```diff
@@ -35,11 +35,18 @@ void factor_vm::handle_safepoint(cell pc) {
   faulting_p = false;
 
   if (atomic::load(&safepoint_fep_p)) {
+    atomic::store(&safepoint_fep_p, false);
```

Comment:
Actually putting it at the top seem to be right and fixes an unrelated bug. If you press t to throw an exception in the low-level debugger the `safepoint_fep_p` variable gets stuck at true state. So if you then run something triggering safepoints like profiling (`[ 3 seconds sleep ] profile`) it breaks.